composer.json

```diff
@@ -13,12 +13,13 @@
         }
     ],
     "require": {
-        "php": ">=5.3.9",
+        "php": ">=5.5.0",
```

Comment:
According to [Jordi's stats](https://seld.be/notes/php-versions-stats-2016-1-edition), PHP <= 5.5 is more around 40% than 80 (in May 2016). Plus it's not like bumping the PHP version requirement is gonna break old releases... As @Taluu it's also the responsibility of library maintainers to push libraries. If there's always a good support for older versions, nobody is going to upgrade.